lib/qrda-import/patient_importer.rb

```diff
@@ -110,11 +110,12 @@ def normalize_references(patient, entry_id_map)
 
           relations_to_add = []
           data_element.relatedTo.each do |related_to|
-            relations_to_add << entry_id_map["#{related_to.value}_#{related_to.namingSystem}"]
+            relations_to_add += entry_id_map["#{related_to.value}_#{related_to.namingSystem}"]
```

Comment:
Not sure I understand this change since it implies that the thing from the entry_id_map goes from a singular object to an array of those objects.

Reply:
The old code resulted in an array that had arrays in it (since the values in entry_id_map are arrays).  The new code combines arrays.